### PR TITLE
Fix PHP8 build

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

**Review by beberlei — APPROVED** [review]

File: tests/Doctrine/Tests/Persistence/ManagerRegistryTest.php [hunk 1]
```diff
@@ -54,7 +55,11 @@ public function testGetManagerForProxyInterface()
     public function testGetManagerForInvalidClass()
     {
         $this->expectException(ReflectionException::class);
-        $this->expectExceptionMessage('Class Doctrine\Tests\Persistence\TestObjectInexistent does not exist');
+        $this->expectExceptionMessage(
+            PHP_VERSION_ID < 80000?
```
Comment (ostrolucky) [inline tests/Doctrine/Tests/Persistence/ManagerRegistryTest.php:59]: This is not an CS violation?

Comment (greg0ire) [reply tests/Doctrine/Tests/Persistence/ManagerRegistryTest.php:59]: Apparently not. Note that we use `doctrine/coding-standard` 6 though.

Comment (ostrolucky) [reply tests/Doctrine/Tests/Persistence/ManagerRegistryTest.php:59]: Space before question mark

Comment (SenseException) [reply tests/Doctrine/Tests/Persistence/ManagerRegistryTest.php:59]: If that space for `?` and `:` doesn't violate the current standard.

Comment (greg0ire) [reply tests/Doctrine/Tests/Persistence/ManagerRegistryTest.php]: Done

**Review by ostrolucky — APPROVED** [review]

**Pull request merged**